Updating BigQuery role info (#8192)
fishtown-analytics/docs.getdbt.com: website/docs/docs/explore/model-query-history.md

     ```
     Without this grant, model query history won't display any data. For more details, view the snowflake docs [here](https://docs.snowflake.com/en/sql-reference/account-usage#enabling-other-roles-to-use-schemas-in-the-snowflake-database). 

> #### BigQuery model query history

Reviewer: 🔥

> The model query history uses metadata from the [`INFORMATION_SCHEMA.JOBS` view](https://docs.cloud.google.com/bigquery/docs/information-schema-jobs) in BigQuery. To access this, the user configured for your production environment must have a proper [IAM role](https://docs.cloud.google.com/bigquery/docs/access-control#bigquery.resourceViewer) for your BigQuery project:

Author: ```suggestion
The model query history uses metadata from the [`INFORMATION_SCHEMA.JOBS` view](https://docs.cloud.google.com/bigquery/docs/information-schema-jobs) in BigQuery. To access this, the user configured for your production environment must have a proper [IAM role](https://docs.cloud.google.com/bigquery/docs/access-control#bigquery.resourceViewer) or permission for your BigQuery project:
```

> The model query history uses metadata from the [`INFORMATION_SCHEMA.JOBS` view](https://docs.cloud.google.com/bigquery/docs/information-schema-jobs) in BigQuery. To access this, the user configured for your production environment must have a proper [IAM role](https://docs.cloud.google.com/bigquery/docs/access-control#bigquery.resourceViewer) or permission for your BigQuery project:

Reviewer: sugg rewording since it read awkwardly to me and i also wasn't clear what 'this' was:
```suggestion
The model query history uses metadata from the [`INFORMATION_SCHEMA.JOBS` view](https://docs.cloud.google.com/bigquery/docs/information-schema-jobs) in BigQuery. To access the metadata, the production environment user must have the correct [IAM role](https://docs.cloud.google.com/bigquery/docs/access-control#bigquery.resourceViewer) or permission to access this data:
```

> - If you're using a BigQuery provided role, we recommend `roles/bigquery.resourceViewer`.

Reviewer: ```suggestion
- If you use a BigQuery provided role, we recommend `roles/bigquery.resourceViewer`.
```

> - If you're using a customer role, it must have the `bigquery.jobs.listAll` permission assigned.

Reviewer: i think we mean custom here, not customer?
```suggestion
- If you use a custom role, ensure it includes the `bigquery.jobs.listAll permission`.
```

Author: EEk! Good catch
